packages/components/src/components/accordion/accordion.stories.ts

```diff
@@ -34,54 +38,52 @@ export const Default = {
 };
 
 /**
- * An accordion item can either be collapsed or open.
+ * Use the attribute `open` to set the initial state of the accordion.
  */
-export const States = {
-  parameters: { controls: { exclude: 'open' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'open' }
-      },
-      args,
-      constants: { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }
-    });
-  }
+
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12">
+      <sd-accordion open summary="Open">Accordion starts open</sd-accordion>
+      <sd-accordion summary="Closed"> Accordion starts closed </sd-accordion>
+    </div>
+  `
 };
 
 /**
- * The summary can have multiple lines.
+ * Use `summary` to provide the text in the accordion header.
  */
-export const SummaryLength = {
-  parameters: { controls: { exclude: 'summary' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'slot',
-          name: 'summary',
-          values: [
-            { value: '<slot slot="summary">Lorem ipsum.</slot>', title: 'short' },
-            {
-              value:
-                '<slot slot="summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</slot>',
-              title: 'long'
-            }
-          ]
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        { type: 'slot', name: 'summary', value: '<div slot="summary" class="slot slot--text">Test</div>`' }
-      ]
-    });
-  }
+
+export const Summary = {
+  name: 'Summary',
+  render: () => html`
+    <div class="grid grid-cols-2">
+      <sd-accordion summary="Summary">Content for summary example</sd-accordion>
+    </div>
+  `
+};
+
+/**
+ * The `summary` slot allows you to use custom html in the summary.
+ */
+
+export const SlottedSummary = {
+  name: 'Slotted Summary',
+  render: () => html`
+    <div class="grid grid-cols-2">
+      <sd-accordion>
+        <div slot="summary"><h4>Custom summary</h4></div>
```

Comment:
```suggestion
        <div slot="summary"><h4>Summary as headline</h4></div>
```

Reply:
Updated after sync with @coraliefeil and @MartaPintoTeixeira